sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.0.0 (2023-11-06)
+## 4.0.0 (2023-11-07)
```

Comment:
does Line 9 (Node 20 support) need to be revised at all?

Reply:
Node 20 support is there. But I think we on the sdk side have dropped the node 16 support. So i think this statement is still valid.